test/functional/blockchain.py

```diff
@@ -40,8 +40,15 @@ def setup_network(self, split=False):
     def run_test(self):
         self._test_gettxoutsetinfo()
         self._test_getblockheader()
+        self._test_getdifficulty()
         self.nodes[0].verifychain(4, 0)
 
+    def _test_getdifficulty(self):
```

Comment:
Can you move the def after the `_test_getblockheader` so it is ordered the same as it is called?